docs/data/material/components/text-fields/InputAdornments.js

```diff
@@ -57,7 +57,9 @@ export default function InputAdornments() {
             endAdornment={
               <InputAdornment position="end">
                 <IconButton
-                  aria-label="toggle password visibility"
+                  aria-label={
+                    showPassword ? 'display the password' : 'hide the password'
```

Comment:
It should be the opposite, as the message should indicate what will happen if you click the button: 

```suggestion
                    showPassword ? 'hide the password' : 'display the password'
```
Same on the other places.

Reply:
Done!